### PR TITLE
Misc changes

### LINKED ISSUE #13075: Elgg 4.0: can we drop elgg_get_filter_tabs
Uses `$context` instead of routes and is only used in specific `content` layouts

### PR DESCRIPTION
**Pull request by jeabakker**
fixes #13075 #13093

### CODE REVIEW

**Review by jdalsem — CHANGES_REQUESTED** [review]

File: mod/groups/views/default/resources/groups/add.php [hunk 1]
```diff
@@ -10,4 +10,5 @@
 
 echo elgg_view_page(elgg_echo('groups:add'), [
 	'content' => $content,
+	'filter_id' => 'groups/edit',
```
[the remark this answers is not in the document]
Comment (jeabakker) [reply mod/groups/views/default/resources/groups/add.php:13]: most add and edit pages use the same form and action, thus i think they should have the same `filter_id`

File: mod/pages/views/default/resources/pages/history.php [hunk 2]
```diff
@@ -39,4 +39,6 @@
 
 echo elgg_view_page($title, [
 	'content' => $content,
+	'filter_id' => 'pages/history',
+	'filter_value' => 'history',
```
[the remark this answers is not in the document]
Comment (jeabakker) [reply mod/pages/views/default/resources/pages/history.php:43]: to know which page you're on as revisions also sets the same `filter_id`

File: mod/thewire/views/default/resources/thewire/thread.php [hunk 3]
```diff
@@ -24,4 +24,6 @@
 
 echo elgg_view_page(elgg_echo('thewire:thread'), [
 	'content' => $content,
+	'filter_id' => 'thewire/view',
```
[the remark this answers is not in the document]
Comment (jeabakker) [reply mod/thewire/views/default/resources/thewire/thread.php]: changed

File: mod/uservalidationbyemail/views/default/resources/uservalidationbyemail/emailsent.php [hunk 4]
```diff
@@ -18,4 +18,6 @@
 	'title' => $title,
 	'content' => elgg_echo('uservalidationbyemail:registerok'),
 	'sidebar' => false,
+	'filter_id' => 'uservalidationbyemail',
+	'fitler_value' => 'emailsent',
```
[the remark this answers is not in the document]
Comment (jeabakker) [reply mod/uservalidationbyemail/views/default/resources/uservalidationbyemail/emailsent.php]: oops

**Review by jdalsem — APPROVED** [review]

**Pull request merged**